lib/src/layouts/awesome/widgets/awesome_camera_gesture_detector.dart

```diff
@@ -87,14 +88,22 @@ class _AwesomeCameraGestureDetector
               GestureRecognizerFactoryWithHandlers<ScaleGestureRecognizer>(
             () => ScaleGestureRecognizer()
               ..onStart = (_) {
-                _previousZoomScale = _zoomScale + 1;
+                _previousZoomValue!.value = _currentZoom + _zoomScaleFactor;
               }
               ..onUpdate = (ScaleUpdateDetails details) {
-                double result = _previousZoomScale * details.scale - 1;
-                if (result < 1 && result > 0) {
-                  _zoomScale = result;
-                  widget.onPreviewScale!.onScale(_zoomScale);
+                final scale = details.scale;
+
+                final result =
+                    _previousZoomValue!.value * scale - _zoomScaleFactor;
```

Comment:
Could you explain what is the role of _zoomScaleFactor here and in onStart?
Maybe @GabrielAraujo could explain ?

Reply:
It was to define the starting point for the zoom, it is particularly used by our application because there are other buttons that set the zoom, no only the pinch gestures